src/main/kotlin/utility/DatabaseFactory.kt

```diff
@@ -22,9 +22,15 @@ object DatabaseFactory {
         transaction {
             SchemaUtils.create(Items, Pokemons, Users)
 
-            Users.insert {
-                it[name] = "Ash"
-                it[pokeDollars] = 10000
+            try {
```

Comment:
So what is this? Shouldn't this be moved to a test class or something?

Reply:
Good point. I removed it since we now can create new users anyways!